docs/UserGuide.md

```diff
@@ -268,10 +267,10 @@ logout
 * Command Phrase are keywords or short phrases that are unique to each Command.
 * Words that are in `UPPER_CASE`{.swift} are parameters that are to be supplied by you.
 * Parameters that are preceded by a prefix must be supplied after that prefix, e.g. `--name DELIVERY_NAME`{.swift}
-  must be given as `--name deliveryName`{.swift} and not `deliveryName`{.swift} or `--name`{.swift}
+  must be given as `--name DELIVERY_NAME`{.swift} and not `DELIVERY_NAME`{.swift} or `--name`{.swift}
```

Comment:
I think this is supposed to be an example of the parameter not the parameter itself

Reply:
right ok changed to actual example